client/GameData/RealmSynergy.cs

```diff
@@ -81,6 +82,7 @@ static RealmSynergy()
                 new SynergyValue("XII", 112001, Value.FF12),
                 new SynergyValue("XIII", 113001, Value.FF13),
                 new SynergyValue("XIV", 114001, Value.FF14),
+                new SynergyValue("Tactics", 150001, Value.FFT),

```

Comment:
Ok great :) I was just trying to get it (obviously having it crash since I'm failing to actually load with the missing values). I'll go ahead and merge it and do some more testing.

If Bacon hasn't modified the DB as needed, let me know and I'll see if I can get to it.